src/file-manager/fm-desktop-icon-view.c

```diff
@@ -244,21 +260,24 @@ desktop_icon_view_property_filter (GdkXEvent *gdk_xevent,
                                    GdkEvent *event,
                                    gpointer data)
 {
-    XEvent *xevent = gdk_xevent;
-    FMDesktopIconView *icon_view;
+    GdkDisplay *display = gdk_display_get_default();
+    if (GDK_IS_X11_DISPLAY (display))
```

Comment:
this again doesn't make much sens, it's an X11-specific function it seems, just call it there only?

Reply:
It is indeed called only when in x11, so marked it as such with a comment and removed the check